manifests/init.pp

```diff
@@ -354,7 +354,13 @@
 #   String. used to set package_checksum for windows installs
 #   Default: undef
 #
-
+# [*windows_pkg_url*]
+#   String.  If specified, override the behavior of computing the package source
+#   URL from windows_repo_prefix and os major release fact.  This parameter is
+#   intended to provide an escape hatch to the situation where the package URL
```

Comment:
docs should not reflect a problem with computing the URL incorrectly and should instead allow a user to use their own URL.